### PR TITLE
Change familial_relationship to be plural (array). Fixes #768.

### LINKED ISSUE #768: Make familial_relationship plural to adhere to Style Guide.
**For which schema is a change/update being suggested?**

I would like to request an update to the `donor_organism.json` schema.

**What should the change/update be?**

- [x] change familial_relationship to familial_relationships since the field takes an array

**What new field(s) need to be changed/added?**

No new fields.

**Why is the change requested?**

Would like to adhere to the Style Guide.

### PR DESCRIPTION
**Pull request by malloryfreeberg**
### Release notes

For donor_organism.json schema:
- change familial_relationship to familial_relationships since the field takes an array

### Reviews requested

- Need 2 Reviewers to approve because this is a major update
- Last to review/approve, merge changes

### CODE REVIEW

File: json_schema/type/biomaterial/donor_organism.json [hunk 1]
```diff
@@ -122,10 +122,11 @@
             "$ref": "module/biomaterial/death.json",
             "user_friendly": "Death conditions"
         },
-        "familial_relationship": {
+        "familial_relationships": {
             "description": "Information about other organisms related to this organism.",
             "type": "array",
             "items": {
+                "type": "object",
```
Comment (daniwelter) [inline json_schema/type/biomaterial/donor_organism.json]: Is this really necessary? Json refs are technically supposed to be the only property in an object as per JSON schema spec as Simon and I discovered yesterday. As a result, ref importing overwrites any sibling fields to $ref (from https://json-schema.org/latest/json-schema-core.html#rfc.section.8.3 - "All other properties in a "$ref" object MUST be ignored."). Technically including other fields is perfectly valid JSON but they will never be considered by a validator.

Comment (malloryfreeberg) [reply json_schema/type/biomaterial/donor_organism.json]: Interesting. I was just explicitly stating type: object to keep everything consistent with the style guide (where type is a required attribute).

Comment (malloryfreeberg) [reply json_schema/type/biomaterial/donor_organism.json]: On second thought, let's keep this clean and not have any other attributes for properties that use $ref. I'll remove it here, make this explicit in the Style Guide, and we can update other fields later.

**Comment by malloryfreeberg** [comment]
@willrockout could you please review and merge this PR? Thanks!

**Pull request merged**